ledger/heavy/handler/handler.go

```diff
@@ -284,3 +290,29 @@ func (h *Handler) handleGotHotConfirmation(ctx context.Context, meta payload.Met
 		logger.Debug("got confirmation: ", confirm.String())
 	}
 }
+
+func (h *Handler) handleGetLightInitialState(ctx context.Context, meta payload.Meta) {
+	startPulse, err := h.StartPulse.PulseNumber()
+	if err != nil {
+		return
+	}
+	if meta.Pulse == startPulse {
+		var IDs []insolar.JetID
+		var drops [][]byte
+		for _, id := range h.JetTree.All(ctx, startPulse) {
```

Comment:
We need to calculate previous pulse with `PulseCalculator` and fetch Jets for it.